vmdb/app/models/mixins/naming_sequence_mixin.rb

```diff
@@ -0,0 +1,17 @@
+module NamingSequenceMixin
+  extend ActiveSupport::Concern
+
+  included do
+    # these are possible naming sequences because there is no constraint to source column
+    has_many :naming_sequences, :as => :resource, :dependent => :destroy, :class_name => "CustomAttribute"
```

Comment:
Normally we use the source column to filter entries in custom_attributes table. Here the source is not fixed but determined by the caller.